Update 100-aws.md (#2632)
kamranahmedse/developer-roadmap: content/roadmaps/102-devops/content/108-cloud-providers/100-aws.md

# AWS

> Amazon Web Services has been the market leading cloud computing platform since 2011, ahead of Azure and Google Cloud. AWS offers over 200 services with data centers located all over the globe, and is the world's most comprehensive cloud plaatform.
> AWS service is an online platform that provides scalable and cost-effectove cloud computing solutions. It is broadly adopted cloud platform that offers seceral on-demand operations like compute power, database storage, content devlivery to helo coorporated scale and grow.

Reviewer: ```suggestion
Amazon Web Services has been the market leading cloud computing platform since 2011, ahead of Azure and Google Cloud. AWS offers over 200 services with data centers located all over the globe.

AWS service is an online platform that provides scalable and cost-effectove cloud computing solutions. It is broadly adopted cloud platform that offers several on-demand operations like compute power, database storage, content delivery and so on.
```
